### PR TITLE
Amazing Mountain Website

### PR DESCRIPTION
**Pull request by zaidansari42**

### CODE REVIEW

**Review by mujeeb100 — CHANGES_REQUESTED** [review]
not a responsive

**Comment by zaidansari42** [comment]
> not a responsive

Please try to View it in a Desktop/Laptop. In mobile desktop version it's not showing, but in Laptop it is responsive as designed

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+  transform: scale(1, 1);
+}
+
+.burger-box:checked ~ .nav-bar a {
+  opacity: 1;
+  transition: opacity 100ms ease-in 400ms;
+}
+
+input.burger-box {
+  display: none;
```
Comment (nehalskdev) [inline styles.css:98]: What is the use of it?

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+
+  height: 65px;
+
+  width: 4em;
+
+  background-color: var(--dark_pink);
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (nehalskdev) [inline styles.css:116]:
```suggestion
  display: grid;
  Place-items: center;
```

**Review by nehalskdev — APPROVED** [review]

File: styles.css (hunk 1 above)
Comment (zaidansari42) [reply styles.css:98]: To make the default view of the checkbox hidden. And then use label as a button for this input.

**Pull request merged**